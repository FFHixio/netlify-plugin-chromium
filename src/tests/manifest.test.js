const fs = require('fs');
const path = require('path');
const YAML = require('yamljs');

describe('NetlifyChromiumPlugin manifest', () => {

  let manifest;

  beforeAll(() => {
    manifest = YAML.parse(fs.readFileSync(path.join(__dirname, '../manifest.yml')).toString());
  });

  it('exists', () => {
    expect(manifest).toBeDefined();
  });

  it('did not change', () => {
    expect(manifest).toMatchSnapshot();
  });

});
