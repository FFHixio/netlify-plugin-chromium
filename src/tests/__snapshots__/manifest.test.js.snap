// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetlifyChromiumPlugin manifest did not change 1`] = `
Object {
  "inputs": Array [
    Object {
      "default": "npm",
      "description": "Package manager to install Chromium with; npm or yarn.",
      "name": "packageManager",
    },
    Object {
      "default": true,
      "description": "If true, sets value of environmental variable CHROME_PATH to location of local copy of Chromium binaries. This change is required by many tools relying on Chromium to be able to find it and launch it successfully (such as Lighthouse).",
      "name": "setChromePathInEnv",
    },
    Object {
      "default": false,
      "description": "If true and Chromium installation finished with failure, whole build will fail. Otherwise, only this plugin fails and the rest of the build proceeds as usual.",
      "name": "failBuildOnError",
    },
  ],
  "name": "netlify-plugin-chromium",
}
`;
